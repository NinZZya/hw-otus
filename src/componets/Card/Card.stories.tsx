import React from "react";
// also exported from "@storybook/react" if you can deal with breaking changes in 6.1
import { Meta } from "@storybook/react/types-6-0";

import { Card } from "./Card";

export default {
  title: "Components/Card",
  component: Card,
} as Meta;

export const EmptyCard: React.FC = () => <Card />;
export const CardWithContent: React.FC = () => <Card>Content</Card>;
