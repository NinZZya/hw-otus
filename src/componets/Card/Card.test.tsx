import React from "react";
import renderer from "react-test-renderer";
import { Card } from "./Card";

it("Card render coorectly", () => {
  const tree = renderer.create(<Card />).toJSON();

  expect(tree).toMatchSnapshot();
});
