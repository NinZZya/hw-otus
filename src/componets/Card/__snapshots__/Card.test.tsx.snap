// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render coorectly 1`] = `
<div
  className="css-1y7yulv"
/>
`;
